emqx/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "fa40ec7e-e8f6-4c4b-a675-31716b23a9df",
+  "app_id": "emqx",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect openmetrics v2 data from EMQX.",
```

Comment:
Please update this description to better describe the types of data being sent in, i.e. Collect performance and health data on MQTT brokers.

Reply:
done :0